assets/src/styles/heroicons.scss

```diff
@@ -127,3 +127,7 @@ i[class^="pb-heroicons-"]::before, i[class*=" pb-heroicons-"]::before {
 .pb-heroicons-wrench::before {
   content: "\f11c";
 }
+
+.pb-heroicons-tablepress::before {
+  content: "\f11d";
```

Comment:
@arzola we need to add a new icon to the dashicons font with this character reference

Reply:
I updated the font family and now includes the table cells icon

Reply:
what's the character reference for the new icon?

Reply:
Found it, nevermind